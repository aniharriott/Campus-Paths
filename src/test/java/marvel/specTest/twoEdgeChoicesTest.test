# Find the alphabetic by edge path between valid characters using twoEdgeChoices

LoadGraph choices twoEdgeChoices.tsv

FindPath noPath a b