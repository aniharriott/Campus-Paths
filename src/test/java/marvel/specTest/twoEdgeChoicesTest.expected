# Find the alphabetic by edge path between valid characters using twoEdgeChoices

loaded graph choices

path from a to b:
a to b via alpha